src/algebra/category/CommRing/basic.lean

```diff
@@ -132,3 +132,68 @@ instance has_forget_to_CommSemiRing : has_forget₂ CommRing CommSemiRing :=
 has_forget₂.mk' (λ R : CommRing, CommSemiRing.of R) (λ R, rfl) (λ R₁ R₂ f, f) (by tidy)
 
 end CommRing
+
+
+namespace ring_equiv
+
+variables {X Y : Type u}
+
+section
+variables [ring X] [ring Y]
+
+/-- Build an isomorphism in the category `Ring` from a `ring_equiv` between `ring`s. -/
+def to_Ring_iso (e : X ≃+* Y) : Ring.of X ≅ Ring.of Y :=
+{ hom := e.to_ring_hom,
+  inv := e.symm.to_ring_hom }
```

Comment:
On the one hand, nothing stops us from having this for `SemiRing` and `CommSemiRing` as well. On the other hand, this means even more code duplication.

Reply:
Let's not do this yet. It's unclear to me that these are particularly useful anyway, except that they might be helpful in experiments to try using `≅` in `Ring` rather than `≃+*`.